Helper/Customer.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * ClassyLlama_AvaTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @copyright  Copyright (c) 2018 Avalara, Inc.
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+namespace ClassyLlama\AvaTax\Helper;
+
+use Magento\Customer\Api\CustomerRepositoryInterface;
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\Exception\LocalizedException;
+
+class Customer extends AbstractHelper
+{
+    /**
+     * @var Config
+     */
+    protected $config;
+
+    /**
+     * @var CustomerRepositoryInterface
+     */
+    protected $customerRepository;
+
+    /**
+     * @param CustomerRepositoryInterface $customerRepository
+     * @param Config                      $config
+     * @param Context                     $context
+     */
+    public function __construct(
+        CustomerRepositoryInterface $customerRepository,
+        Config $config,
+        Context $context
+    )
+    {
+        parent::__construct($context);
+
+        $this->config = $config;
+        $this->customerRepository = $customerRepository;
+    }
+
+    /**
+     * @param int $customerId
+     *
+     * @return \Magento\Customer\Api\Data\CustomerInterface|null
+     */
+    protected function getCustomerById($customerId)
+    {
+        $customer = null;
+
+        try {
+            $customer = $this->customerRepository->getById($customerId);
```

Comment:
@LordZardeck - Two questions:

1) Why are we wrapping the customer repository getById method in a helper rather than just using it directly where this is called?
2) Why is it appropriate for these catches to be empty?  (Seeing a lot of those.  It's suspect to me that it would very frequently be appropriate to catch an exception and do nothing about it.)